Decisions.Equifax/PrequalificationOfOne/PreQualificationOfOneStep.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Decisions.Equifax.ConsumerCreditReport.Request;
+using Decisions.Equifax.PreQualificationOfOne.Response;
+using DecisionsFramework;
+using DecisionsFramework.Design.Flow;
+using DecisionsFramework.ServiceLayer;
+
+namespace Decisions.Equifax.PrequalificationOfOne
+{
+    [AutoRegisterMethodsOnClass(true, "Integration/Equifax/PreQualification")]
+    public class PreQualificationOfOneStep 
+    {
+        /// <summary>
+        /// Step: Get Pre-qualification of One (Integration/Equifax/Consumer Credit Report/Pre-qualification of One)
```

Comment:
This comment isn't necessary, but should be correct if it's there.

Reply:
Corrected